src/main/java/net/dv8tion/jda/core/entities/impl/UserImpl.java

```diff
@@ -92,6 +92,11 @@ public String getEffectiveAvatarUrl()
         return getAvatarUrl() == null ? getDefaultAvatarUrl() : getAvatarUrl();
     }
 
+    @Override
+    public String getAsTag()
+    {
+        return String.format("%#s", this);
```

Comment:
Using a formatter just for this is pretty overkill

Reply:
Yea... i would just copy the code of the format method and then replace the one in the format method by a getAsTag() method call

Reply:
i mean i guess we use `String.format()` in `UserImpl#formatTo()` too... https://github.com/DV8FromTheWorld/JDA/blob/a7bfefe45f35d96b9d5b313f8b55b6ea926a873e/src/main/java/net/dv8tion/jda/core/entities/impl/UserImpl.java#L240-L256

Reply:
I think it would be reasonable to replace these with `getAsTag` similar to the `getAsMention` here

Reply:
I'm unsure what to do with the upper flag, or should I just ditch that?

Reply:
`toUpperCase()`?

Reply:
🤦